models/ConfigureForm.php

```diff
@@ -26,7 +29,8 @@ public function rules()
         return [
             [['enabledPages'], 'in', 'range' => array_keys(Page::getPages())],
             [['defaultLanguage'], 'in', 'range' => array_keys(Yii::$app->i18n->getAllowedLanguages())],
-            [['showAgeCheck'], 'boolean']
+            [['showAgeCheck'], 'boolean'],
+            [['defaultAge'], 'string'],
```

Comment:
Seems I also forgot to add `[['defaultAge'], 'validateAge'],` rules here :facepalm: